src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -336,7 +331,7 @@ public Response getPoliciesForCustomer() {
   @APIResponse(responseCode = "404", description = "No policies found for customer")
   @APIResponse(responseCode = "403", description = "Individual permissions missing to complete action")
   @APIResponse(responseCode = "200", description = "PolicyIds found", content =
-                 @Content(schema = @Schema(implementation = ResponseListOfUUIDS.class)))
+                 @Content(schema = @Schema(type = SchemaType.ARRAY, implementation = UUID.class)))
```

Comment:
I think we have some other places like this.
Does this change not apply there as well?

Reply:
Probably, I'll check.

Reply:
Other places already use the `type = SchemaType.Array, implementation = UUID.class` stuff, and there is no other raw array that needs a change like this.